bench/utils.py

```diff
@@ -687,9 +687,11 @@ def set_git_remote_url(git_url, bench_path='.'):
 	if os.path.exists(os.path.join(app_dir, '.git')):
 		exec_cmd("git remote set-url upstream {}".format(git_url), cwd=app_dir)
 
-def run_playbook(playbook_name):
+def run_playbook(playbook_name, extra_vars=None):
 	if not find_executable('ansible'):
 		print "Ansible is needed to run this command, please install it using 'pip install ansible'"
 		sys.exit(1)
 	args = ['ansible-playbook', '-c', 'local', playbook_name]
+	if extra_vars:
+		args.extend(['--extra-vars=' + json.dumps(extra_vars)])
```

Comment:
Keeping it consistent with `install.py` args.extend(['-e', get_extra_vars_json(extra_vars)])